elm/sandbox/server/walker/walk_creator.py

```diff
@@ -34,38 +35,49 @@ def serialize_walker(self):
             joints.append((j[0], j[1]))
         for m in self.muscles:
             if m.type == "distance":
-                muscles.append([self.joint_index(m.j0), self.joint_index(m.j1), {"type": m.type}])
+                muscles.append(
+                    [self.joint_index(m.j0), self.joint_index(m.j1), {"type": m.type}]
+                )
             elif m.type == "muscle":
-                muscles.append([self.joint_index(m.j0), self.joint_index(m.j1),
-                                {"type": m.type, "amplitude": m.amplitude, "phase": m.phase}])
+                muscles.append(
+                    [
+                        self.joint_index(m.j0),
+                        self.joint_index(m.j1),
+                        {"type": m.type, "amplitude": m.amplitude, "phase": m.phase},
+                    ]
+                )
         return {"joints": joints, "muscles": muscles}
 
     def serialize_walker_sodarace(self):
         walker_dict = {
             "useLEO": True,
-            "nodes": [
-            ],
-            "connections": [
-            ],
+            "nodes": [],
+            "connections": [],
         }
         for j in self.joints:
-            walker_dict["nodes"].append({
-                "x": j[0],
-                "y": j[1],
-            })
+            walker_dict["nodes"].append(
+                {
+                    "x": j[0],
+                    "y": j[1],
```

Comment:
Here `j[0], j[1]` are still wrapped with numpy. They are not serializable when passing through flask endpoint.
`j[0].item()` and `j[1].item()` seem to fix it.